core/src/main/java/kafka/server/SharePartitionManager.java

```diff
@@ -161,7 +167,38 @@ public ShareFetchContext newContext(Map<TopicIdPartition,
     // TODO: Define share session class.
     public static class ShareSession {
 
+        private final String id;
+        private final boolean privileged;
+        private final ImplicitLinkedHashCollection<SharePartitionManager.CachedPartition> partitionMap;
+        private final long creationMs;
+        private final long lastUsedMs;
+        private final int epoch;
+
+        /**
+         * The share session.
+         * Each share session is protected by its own lock, which must be taken before mutable
+         * fields are read or modified.  This includes modification of the share session partition map.
+         *
+         * @param id                 The unique share session ID.
+         * @param privileged         True if this share session is privileged.  Sessions crated by followers
```

Comment:
We do not need the concept of privileged for share sessions as far as I can see. Please remove.